src/devices/bus/spectrum/mface.cpp

```diff
@@ -191,13 +305,18 @@ void spectrum_mface1_device::pre_opcode_fetch(offs_t offset)
 uint8_t spectrum_mface1_device::iorq_r(offs_t offset)
 {
 	uint8_t data = m_exp->iorq_r(offset);
+	
+	if (ioport("CONFIG")->read() & 0x01)  // disable switch
```

Comment:
Please use a `required_ioport` for this - you shouldn't have tag map lookups in the hot path.

Reply:
Ok I will change that...